Makefile

```diff
@@ -1,16 +1,38 @@
+.PHONY: clean quality requirements validate
+
 clean:
-	find . -name '*.pyc' -delete
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +	
 	coverage erase
 	rm -rf coverage htmlcov
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
 
 quality:
 	tox -e quality
 
+
 requirements:
-	pip install -r test_requirements.txt
+	@echo "What up"
```

Comment:
i think that needs to be removed.

Reply:
yes, still testing things out, haven't had time to clean up my changes yet, one of the travis tests is still failing

Reply:
Just removed random output

Reply:
lol, thanks for pointing that out, I had completely forgotten